src/services/ui/src/features/package/package-activity/index.tsx

```diff
@@ -12,6 +12,41 @@ import * as Table from "@/components";
 import { BLANK_VALUE } from "@/consts";
 import { usePackageActivities, useAttachmentService } from "./hook";
 import { Link } from "@/components/Routing";
+import { attachmentTitleMap } from "shared-types";
+
+// id, attachments, hook
+const AttachmentDetails: FC<{
```

Comment:
YAY! Maybe bolster w/ a unit test? the only real logic is measuring the attachmentLabel so maybe it's extra...idk 🤷🏼‍♀️

Reply:
I think for now the typing will check the keys are correct but we could defs keep this in mind for the future :)